classes/Language.php

```diff
@@ -1188,15 +1188,15 @@ public static function countActiveLanguages($id_shop = null)
         return static::$countActiveLanguages[$id_shop];
     }
 
-    public static function downloadAndInstallLanguagePack($iso, $version = _PS_VERSION_, $params = null, $install = true)
+    public static function downloadAndInstallLanguagePack($iso, $params = null, $install = true)
```

Comment:
Still the same feedback: I don't think we should remove the optional parameters when they are not in the last position I get that this `$version` parameter is useless and was never used but still we can't change the other parameters' order

Its even more true for this function because the following parameters are critical for the install to work correctly so we can't mess with the initial signature